### PR TITLE
Add cli command for message sign/verify

### PR DESCRIPTION
**Pull request by coderofstuff**
New CLI command: `message`

Subcommands:
- sign <kaspa_address> - Sign a message with the private key that matches the given address. Prompts for message.
- verify <kaspa_address> <signature> - Verify the signature against the message and kaspa_address. Prompts for message.

### CODE REVIEW

File: cli/src/modules/message.rs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            AccountKind::Bip32 => {
+                let (wallet_secret, payment_secret) = ctx.ask_wallet_secret(Some(&account)).await?;
+                let keydata = account.prv_key_data(wallet_secret).await?;
+                let mut keys = HashMap::new();
+                let account = account.clone().as_derivation_capable().expect("expecting derivation capable");
+
+                let (receive, change) = account.derivation().addresses_indexes(&[&kaspa_address])?;
+                let private_keys = account.create_private_keys(&keydata, &payment_secret, &receive, &change)?;
+                for (address, private_key) in private_keys {
+                    keys.insert(address.clone(), private_key.secret_bytes());
```
Comment (biryukovmaxim) [inline cli/src/modules/message.rs:142]: Instead of insertion you can compare addresses and return it directly

Comment (coderofstuff) [reply cli/src/modules/message.rs]: Fixed in [69020cb](https://github.com/kaspanet/rusty-kaspa/pull/280/commits/69020cbc0ef03a807a05554acb96ffca57ebb281)

File: cli/src/modules/message.rs [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            return self.display_help(ctx, argv).await;
+        }
+
+        match argv.get(0).unwrap().as_str() {
+            "sign" => {
+                if argv.len() < 2 {
+                    return self.display_help(ctx, argv).await;
+                }
+
+                let kaspa_address = argv.get(1).unwrap().as_str();
```
Comment (biryukovmaxim) [inline cli/src/modules/message.rs:38]: If there's no argument, you can try to get current active account or ask to select it

File: cli/src/modules/message.rs [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        let pm = PersonalMessage(message);
+        let privkey = self.get_address_private_key(&ctx, kaspa_address).await?;
+
+        let sig_result = sign_message(&pm, &privkey);
+
+        match sig_result {
+            Ok(signature) => {
+                let sig_hex = faster_hex::hex_string(signature.as_slice());
+                tprintln!(ctx, "Signature: {}", sig_hex);
+            }
```
Comment (biryukovmaxim) [inline cli/src/modules/message.rs:94]:
```
Ok(())
```
Then you will be able to remove `return` word in the second condition branch and 'Ok(())' below

**Review by aspect — APPROVED** [review]
Everything looks good.

**Pull request merged**